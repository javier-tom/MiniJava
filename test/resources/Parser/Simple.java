class Main {
    public static void main(String[] args) {
        x = Something.nothing(1);
    }
}

class Something {
    public int nothing(int arg) {
        return arg;
    }
}