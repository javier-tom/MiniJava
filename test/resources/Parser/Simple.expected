Program
  MainClass Main (line 1)
    Assign x (line 3)
      new value:
        Call nothing (line 3)
          on class:
            Something (line 3)
          parameters:
            1 (line 3)

  Class Something (line 7)
    fields:
    MethodDecl nothing (line 8)
      returns int
      params:
        int arg
      variables:
      Return
        arg (line 9)